tests/integration/vault_kv_requirer_operator/charmcraft.yaml

```diff
@@ -18,6 +18,7 @@ storage:
   certs:
     type: filesystem
     minimum-size: 5M
+    location: /var/snap/vault/common/certs
```

Comment:
What is the value in setting this?

Reply:
Nothing, I was getting an error with the `storage-attached` hook, which turned out to be unrelated, but I update this to match what we do in the charm while debugging.

Reply:
Im a little anxious to set something explicit if we don't need it, but I don't have a specific reason. I guess if the structure of where snaps store things change, for example.

Reply:
Removed :)